masterlist.yaml

```diff
@@ -188,6 +188,11 @@ common:
         text: 'Delete or deactivate %1%.'
       - lang: ru
         text: 'Не забудьте отключить или удалить %1%.'
+  - &redundantPluginX
+    type: warn
+    content:
+      - lang: en
+        text: 'This plugin is redundant when using %1% and should be deactivated or deleted.'
```

Comment:
Looking over the uses of this, it seems they would all be appropriate for either `&alreadyInX` or `&useOnlyOneX`. Are there any you feel wouldn't fit into either of these?